famstories/index.html

```diff
@@ -34,8 +34,8 @@
         <a href="/bernedoodle">Bernedoodle Info</a>
         <a href="/breed">AussieDoodle Info</a>
         <a href="/about">About us</a>
-        <a href="/current" class="current">Current Litter</a>
-        <a href="famstories">Reviews</a>
+        <a href="/current">Current Litter</a>
+        <a href="famstories"class="current">Reviews</a>
```

Comment:
There needs to be a space before the word `class` here
```suggestion
        <a href="famstories" class="current">Reviews</a>
```